clients/da-vinci-client/src/main/java/com/linkedin/davinci/listener/response/ReadResponse.java

```diff
@@ -75,12 +74,12 @@ public void addReadComputeDeserializationLatency(double latency) {
     this.readComputeDeserializationLatency += latency;
   }
 
-  public void setOptionalKeySizeList(Optional<List<Integer>> optionalKeySizeList) {
-    this.optionalKeySizeList = optionalKeySizeList;
+  public void setKeyListSize(IntList keyListSize) {
+    this.keyListSize = keyListSize;
   }
 
-  public void setOptionalValueSizeList(Optional<List<Integer>> optionalValueSizeList) {
-    this.optionalValueSizeList = optionalValueSizeList;
+  public void setValueListSize(IntList valueListSize) {
+    this.valueListSize = valueListSize;
```

Comment:
Can we rename those functions and variables to `keySizeList` and `valueSizeList`?